pallets/parachain-staking/src/weights.rs

```diff
@@ -177,12 +177,15 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 			.saturating_add(T::DbWeight::get().writes(7 as Weight))
 	}
 	fn active_on_initialize(x: u32, y: u32) -> Weight {
-		(10_177_000 as Weight)
-			// Standard Error: 0
-			.saturating_add((24_000 as Weight).saturating_mul(x as Weight))
-			// Standard Error: 0
-			.saturating_add((1_000 as Weight).saturating_mul(y as Weight))
-			.saturating_add(T::DbWeight::get().reads(2 as Weight))
+		(0 as Weight)
+			// Standard Error: 299_000
+			.saturating_add((208_550_000 as Weight).saturating_mul(x as Weight))
+			// Standard Error: 27_000
+			.saturating_add((15_580_000 as Weight).saturating_mul(y as Weight))
+			.saturating_add(T::DbWeight::get().reads(26 as Weight))
+			.saturating_add(T::DbWeight::get().reads((4 as Weight).saturating_mul(x as Weight)))
+			.saturating_add(T::DbWeight::get().writes(16 as Weight))
+			.saturating_add(T::DbWeight::get().writes((4 as Weight).saturating_mul(x as Weight)))
```

Comment:
these are the numbers I got the second time I ran it on the benchmarking server:
```rust
(0 as Weight)
            // Standard Error: 283_000
            .saturating_add((201_041_000 as Weight).saturating_mul(x as Weight))
            // Standard Error: 26_000
            .saturating_add((14_932_000 as Weight).saturating_mul(y as Weight))
            .saturating_add(T::DbWeight::get().reads(26 as Weight))
            .saturating_add(T::DbWeight::get().reads((4 as Weight).saturating_mul(x as Weight)))
            .saturating_add(T::DbWeight::get().writes(16 as Weight))
            .saturating_add(T::DbWeight::get().writes((4 as Weight).saturating_mul(x as Weight)))
```

Reply:
They're pretty similar, slightly better in your run. So... do we prefer safety or performance here? :thinking:

Reply:
3rd run:
```suggestion
		(0 as Weight)
            // Standard Error: 296_000
            .saturating_add((202_812_000 as Weight).saturating_mul(x as Weight))
            // Standard Error: 27_000
            .saturating_add((15_031_000 as Weight).saturating_mul(y as Weight))
            .saturating_add(RocksDbWeight::get().reads(26 as Weight))
            .saturating_add(RocksDbWeight::get().reads((4 as Weight).saturating_mul(x as Weight)))
            .saturating_add(RocksDbWeight::get().writes(16 as Weight))
            .saturating_add(RocksDbWeight::get().writes((4 as Weight).saturating_mul(x as Weight)))
```

Reply:
> They're pretty similar, slightly better in your run. So... do we prefer safety or performance here? 🤔

I thought we had already established that the first run always has the highest numbers, but this raises a good question: should we prefer the worst run (first run) or subsequent runs when the weights always plateau at slighly better than the first run?

Reply:
IMO, safety is more important initially. We'll have plenty of opportunity to optimize down the road. But it's probably a good idea to scrutinize the results either way when we want "official numbers".

That said, they're similar enough here that I don't have a strong opinion.